app/build.gradle

```diff
@@ -32,8 +35,11 @@ android {
 }
 
 dependencies {
+    def fragment_version = "1.3.2"
 
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
```

Comment:
and now
add and use
https://github.com/terrakok/Cicerone
https://developer.android.com/topic/libraries/view-binding (already enabled in this PR)
https://github.com/kirich1409/ViewBindingPropertyDelegate